cluster/gce/gci/configure-helper.sh

```diff
@@ -893,7 +893,8 @@ function create-kubelet-kubeconfig() {
     exit 1
   fi
   echo "Creating kubelet kubeconfig file"
-  cat <<EOF >/var/lib/kubelet/bootstrap-kubeconfig
+  if [[ "${CREATE_BOOTSTRAP_KUBECONFIG:-true}" == "true" ]]; then
```

Comment:
I'd suggest sourcing this from the metadata server in a new attribute. It would give us a bit more flexibility later. What do you think?